sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSqlJavaUdfTypeTest.java

```diff
@@ -181,293 +181,268 @@ private void runUdfTypeTest(String query, Object result, Schema.TypeName typeNam
+  public void testMinPosFloat64Input() {
     runUdfTypeTest(
-        "SELECT test(float64_min_pos) FROM table;",
+        "SELECT test_float64(float64_min_pos) FROM table;",
         2.2250738585072014e-308,
-        Schema.TypeName.DOUBLE,
-        doubleUdf);
+        Schema.TypeName.DOUBLE);
   }
 
   @Test
   @Ignore(
       "+inf is implemented as a ZetaSQL builtin function, so combining it with a UDF requires Calc splitting (BEAM-12009).")
-  public void testPosInfFloat64Literal() throws NoSuchMethodException {
+  public void testPosInfFloat64Literal() {
     runUdfTypeTest(
-        "SELECT test(CAST('+inf' AS FLOAT64));",
+        "SELECT test_float64(CAST('+inf' AS FLOAT64));",
         Double.POSITIVE_INFINITY,
-        Schema.TypeName.DOUBLE,
-        doubleUdf);
+        Schema.TypeName.DOUBLE);
   }
 
   @Test
-  public void testPosInfFloat64Input() throws NoSuchMethodException {
+  public void testPosInfFloat64Input() {
     runUdfTypeTest(
-        "SELECT test(float64_inf) FROM table;",
+        "SELECT test_float64(float64_inf) FROM table;",
         Double.POSITIVE_INFINITY,
-        Schema.TypeName.DOUBLE,
-        doubleUdf);
+        Schema.TypeName.DOUBLE);
   }
 
   @Test
   @Ignore(
       "-inf is implemented as a ZetaSQL builtin function, so combining it with a UDF requires Calc splitting (BEAM-12009).")
-  public void testNegInfFloat64Literal() throws NoSuchMethodException {
+  public void testNegInfFloat64Literal() {
     runUdfTypeTest(
-        "SELECT test(CAST('-inf' AS FLOAT64));",
+        "SELECT test_float64(CAST('-inf' AS FLOAT64));",
         Double.NEGATIVE_INFINITY,
-        Schema.TypeName.DOUBLE,
-        doubleUdf);
+        Schema.TypeName.DOUBLE);
   }
 
   @Test
-  public void testNegInfFloat64Input() throws NoSuchMethodException {
+  public void testNegInfFloat64Input() {
     runUdfTypeTest(
-        "SELECT test(float64_neg_inf) FROM table;",
+        "SELECT test_float64(float64_neg_inf) FROM table;",
         Double.NEGATIVE_INFINITY,
-        Schema.TypeName.DOUBLE,
-        doubleUdf);
+        Schema.TypeName.DOUBLE);
   }
 
   @Test
   @Ignore(
       "NaN is implemented as a ZetaSQL builtin function, so combining it with a UDF requires Calc splitting (BEAM-12009).")
-  public void testNaNFloat64Literal() throws NoSuchMethodException {
+  public void testNaNFloat64Literal() {
     runUdfTypeTest(
-        "SELECT test(CAST('NaN' AS FLOAT64));", Double.NaN, Schema.TypeName.DOUBLE, doubleUdf);
+        "SELECT test_float64(CAST('NaN' AS FLOAT64));", Double.NaN, Schema.TypeName.DOUBLE);
   }
 
   @Test
-  public void testNaNFloat64Input() throws NoSuchMethodException {
+  public void testNaNFloat64Input() {
     runUdfTypeTest(
-        "SELECT test(float64_nan) FROM table;", Double.NaN, Schema.TypeName.DOUBLE, doubleUdf);
+        "SELECT test_float64(float64_nan) FROM table;", Double.NaN, Schema.TypeName.DOUBLE);
   }
-
-  // TODO(ibzib) Test that dates and times are rejected.
```

Comment:
I'm fine with deleting these (I'm not a big fan of TODOs) but wanted to confirm these issues still remain?

Reply:
These TODOs were just for my short-term personal use, long-term they're better off tracked as JIRAs: BEAM-12106, BEAM-12108, BEAM-12086, BEAM-12087, BEAM-11990.

Reply:
I also created a JIRA label `zetasql-java-udf` since I was starting to have trouble finding these.